### PR TITLE
added Implementation Guide for easy-subscribe

### PR DESCRIPTION
**Pull request by ankush-notifi**
Added Implementation Guide for Easy-Subscribe

https://notifi.atlassian.net/browse/MVP-5477

### CODE REVIEW

**Review by eric-notifi — COMMENTED** [review]
LGTM, informative. 👍
Just wondering shall we also need to add a copy version under `/docs/` even it will not be reflected on page.
I am not sure why we have a copy version under `/docs/` and what is the purpose of it. It will be appreciated anyone who knows more about it to elaborate it more. (Maybe  even better to make a note in Readme.
CC: @mayuranganesathas @nimesh-notifi

**Comment by mayuranganesathas** [comment]
> LGTM, informative. 👍 Just wondering shall we also need to add a copy version under `/docs/` even it will not be reflected on page. I am not sure why we have a copy version under `/docs/` and what is the purpose of it. It will be appreciated anyone who knows more about it to elaborate it more. (Maybe even better to make a note in Readme. CC: @mayuranganesathas @nimesh-notifi

+1, the /next path needs to be updated

File: versioned_docs/version-Release/alert-subscribe/easy-subscribe/index.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
+
+# Easy Subscribe
```
Comment (mayuranganesathas) [inline versioned_docs/version-Release/alert-subscribe/easy-subscribe/index.md:2]: FIX: or manage notification preferences. 

FIX:  Users can subscribe using their email address and have the option to receive notifications via a Telegram bot.

File: versioned_docs/version-Release/alert-subscribe/easy-subscribe/index.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+With the **Easy Subscription** feature, users can subscribe to Community Manager topics directly, without needing to create a Notifi account, sign up on the Notifi React Card, manage preferences, or sign their wallet. Users can subscribe using their email and can opt to subscribe to the Telegram bot.
+
+# Creating an Easy Subscribe Card Config
+
+To use the **Easy Subscribe** feature, a card configuration must be created. This configuration sets up Easy Subscription campaign and provides the necessary endpoints, allowing you to directly use these URLs to join the Telegram bot and send email addresses for subscribing to Community Manager topics.
```
Comment (mayuranganesathas) [inline versioned_docs/version-Release/alert-subscribe/easy-subscribe/index.md]: fix: a respective Easy Subscribe card configuration must be created.
fix: sets up an Easy Subscription campaign

fix: necessary endpoints: allowing you to directly.......

fix: and send to email addresses

File: versioned_docs/version-Release/alert-subscribe/easy-subscribe/index.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+
+Check out these sections to set up your Telegram bot and Email domain:
+
+- [Set up a Telegram bot](../target-setup/tg-bot)
+- [Set up a custom email domain](../target-setup/email-domain)
+
+:::
+
+### Step 6: Add Templates
+These templates will be sent to users who subscribe to topics using the **Easy Subscribe** config. They can be in the form of a greeting or confirmation email/Telegram message, which the users will receive on their respective platforms upon subscribing to topics.
```
Comment (mayuranganesathas) [inline versioned_docs/version-Release/alert-subscribe/easy-subscribe/index.md]: Email/Telegram

**Review by mayuranganesathas — APPROVED** [review]
Some comments but LGTM

**Review by mayuranganesathas — APPROVED** [review]

**Pull request merged**